src/main/webapp/app/pages/curation/collapsible/ReviewCollapsible.tsx

```diff
@@ -92,7 +93,9 @@ export const ReviewCollapsible = (props: IReviewCollapsibleProps) => {
           icon={faCheck}
           color={SUCCESS}
           onClick={() => {
-            props.handleAccept(props.hugoSymbol, [props.baseReviewLevel as ReviewLevel], props.isGermline);
+            props.handleAccept(props.hugoSymbol, [props.baseReviewLevel as ReviewLevel], props.isGermline).catch(error => {
```

Comment:
Should the prop ever be something other than a ReviewLevel object?

Reply:
`props.baseReviewLevel` can be MetaReviewLevel or ReviewLevel. We still render the MetaReviewLevel in this component.